rasa/core/brokers/pika.py

```diff
@@ -8,6 +8,7 @@
 from typing import Deque, Dict, Optional, Text, Union, Any, List, Tuple
 
 import aio_pika
+from aiormq.exceptions import ChannelNotFoundEntity
```

Comment:
I think we can revert the changes in this file now right?

Reply:
Done 😄

Reply:
there's still a couple more in the file 😄

Reply:
oups, found it - all removed now I hope 😅